docs/documentation/head/query.md

```diff
@@ -96,28 +97,40 @@ Changing code to cursor mode is as simple as setting the fetch size of the
 all rows to be cached (the default behaviour).
 
 ```java
+import static java.sql.ResultSet.*;
 // make sure autocommit is off
 conn.setAutoCommit(false);
-Statement st = conn.createStatement();
+Statement st = conn.createStatement(TYPE_FORWARD_ONLY, CONCUR_READ_ONLY, CLOSE_CURSORS_AT_COMMIT);
```

Comment:
`CLOSE_CURSORS_AT_COMMIT` is a default value for statement holdability. This example looks too complicated

Reply:
All of those values are the defaults. The intent was to explicitly show that the required settings were used. I'm reworking the example to try and make that more clear.